### PR TITLE
Allow multiple title pages, show current title

### PR DESCRIPTION
**Pull request by NoneNoname**

### CODE REVIEW

File: BondageClub/Screens/Character/Title/Title.js [hunk 1]
```diff
@@ -180,22 +184,25 @@ function TitleClick() {
...
-			}
-			X = X + 450;
-			if (X > 1500) {
-				X = 130;
-				Y = Y + 90;
-			}
+	let X = 130;
+	let Y = 250;
+	for (let T = TitleOffset; T < TitleOffset + TitlePerPage && T < TitleListFiltered.length; T++) {
+		if ((MouseX >= X) && (MouseX <= X + 400) && (MouseY >= Y) && (MouseY <= Y + 65)) {
```
Comment (elliesec) [inline BondageClub/Screens/Character/Title/Title.js:196]: Probably worth refactoring this to use `MouseIn` while we're making changes here.
```suggestion
		if (MouseIn(X, Y, 400, 65)) {
```

Comment (Sekkmer) [reply BondageClub/Screens/Character/Title/Title.js]: sure, fixed the exit button as well then

Comment (NoneNoname) [reply BondageClub/Screens/Character/Title/Title.js]: sure, fixed the exit button as well

**Comment by Ben987** [comment]
Good idea!  Thanks for that one :)

**Pull request merged**